R/parameters.R

```diff
@@ -334,6 +336,8 @@ get_parameters <- function(overrides = list(), parasite = "falciparum") {
       prevalence_rendering_max_ages = 10 * 365,
       incidence_rendering_min_ages = numeric(0),
       incidence_rendering_max_ages = numeric(0),
+      patent_incidence_rendering_min_ages = numeric(0),
+      patent_incidence_rendering_max_ages = numeric(0),
```

Comment:
This is not consistent with the documentation. We probably want to change the documentation? Since I imagine changing this would break the run.

Reply:
Documentation added. Everything seems fine...?